scripts/runalltests.sh

```diff
@@ -0,0 +1,37 @@
+workdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"/testslog
```

Comment:
I usually use `workdir="$(dirname "$0")/testslog"` - why would you use the extra `pwd` step?